src/demo/index.html

```diff
@@ -628,6 +638,29 @@ <h1>Symphony Electron API Demo</h1>
             activityTimer = startActivityInterval();
         };
 
+        /**
+         * Get CPU usage
+         */
+        const cpuMessurement = document.getElementById('button-get-cpu-usage');
+
+        cpuMessurement.addEventListener('click', () => {
+            console.log('Start messure CPU !!!');
```

Comment:
typo: please rename `messure` to `measure`

Reply:
fixed